configs/azure_c_shared_utilityConfig.cmake

```diff
@@ -3,10 +3,13 @@
 
 include("${CMAKE_CURRENT_LIST_DIR}/azure_c_shared_utilityTargets.cmake")
 
+include(CMakeFindDependencyMacro)
+set(CURL_DIR CMake)
+find_dependency(CURL CONFIG)
+
```

Comment:
IMO you don't need changes to azure_c_shared_utilityConfig.cmake as long as you are linking to `${CURL_LIBRARIES}`. vcpkg needs to fix the variable content.

If you would link to `CURL::libcurl`, you would only need:
~~~cmake
include(CMakeFindDependencyMacro)
find_dependency(CURL)
~~~